boxsdk/util/enum.py

```diff
@@ -82,8 +82,11 @@ def __members__(cls):
         return members
 
     def __contains__(cls, member):
-        if super(ExtendableEnumMeta, cls).__contains__(member):
-            return True
+        try:
+            if super(ExtendableEnumMeta, cls).__contains__(member):
```

Comment:
This is added due to following changes in __constains__ method of Enum in Python
in Python 3.9:
`def __contains__(cls, member):
        if not isinstance(member, Enum):
            raise TypeError(
                "unsupported operand type(s) for 'in': '%s' and '%s'" % (
                    type(member).__qualname__, cls.__class__.__qualname__))
        return isinstance(member, cls) and member._name_ in cls._member_map_`

in Python 3.6:

`def __contains__(cls, member):
        return isinstance(member, cls) and member._name_ in cls._member_map_`